WarpLib/WorkerWrapper.cs

```diff
@@ -61,6 +61,11 @@ public WorkerWrapper(int deviceID, bool silent = false, bool attachDebugger = fa
             }
             else
             {
+                // named pipes on linux don't work well on remote filesystems
+                // https://github.com/warpem/warp/issues/28#issuecomment-2197168677
+                string PipeDirectory = Environment.GetEnvironmentVariable("TMPDIR");
+                PipeName = $"{PipeDirectory}{PipeName}";
```

Comment:
You'll also want to include [Path.DirectorySeparatorChar](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-8.0) (or something similar) between `TMPDIR` and `PipeName` since `TMPDIR` is typically set to `/tmp` (without a `/` as a suffix).  Without a path separator, you might get things like `/tmpWarpWorkerPipe_1_3073889_2059106402` instead of the desired `/tmp/WarpWorkerPipe_1_3073889_2059106402`.  

It never hurts to have multiple path separators, so even if the value of `TMPDIR` ends with `/` adding an extra `/` should not be problematic (i.e., `/tmp//WarpWorkerPipe_1_3073889_2059106402` is resolved in the same way as `/tmp/WarpWorkerPipe_1_3073889_2059106402`.

Reply:
You also might want to try [Path.Combine](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-8.0) here.

Reply:
whoops! Good catch - thanks